src/Shaders/VertexShader.hlsl

```diff
@@ -5,8 +5,13 @@ cbuffer CameraMatrices : register(b0) {
     float4x4 projectionMatrix;  // 16 floats
 };
 
+#define MAX_INSTANCE_SIZE 1000
```

Comment:
is this limitation going to be a problem for when we have fluid with many particles? do we need one entire matrix per particle or just a position? also, why do we no longer use SRV and now use CBV